src/datachain/remote/studio.py

```diff
@@ -239,6 +240,52 @@ def _send_request(
 
         return Response(data, ok, message, response.status_code)
 
+    @retry_with_backoff(retries=3, errors=(HTTPError, Timeout))
```

Comment:
I would be really careful with retries for requests sent with `POST` method by default 👀
May be we should add `retry` decorator in-place, for methods we are sure it is safe to retry (like `get_jobs`) and not add it by default for other methods (like `upload_file`, which can in theory fail with timeout error but file will be uploaded successfully)? What do you think?